### PR TITLE
DMP-2319-Courthouse-Details-View-Users-Tab 

### PR DESCRIPTION
**Pull request by bpires2**
### Jira link (if applicable)

[DMP-2319](https://tools.hmcts.net/jira/browse/DMP-2319)

### Change description ###

- Courthouse record users page

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Comment by DavidTMann** [comment]
@davet1985 Is it likely to have no users against a courthouse in Prod? At the moment we just show a blank screen which confused me (on stub)

**Comment by davet1985** [comment]
> @davet1985 Is it likely to have no users against a courthouse in Prod? At the moment we just show a blank screen which confused me (on stub)

Yes, this can happen as new courthouses can be created, or all users could be removed from a courthouse.

**Comment by davet1985** [comment]
> > @davet1985 Is it likely to have no users against a courthouse in Prod? At the moment we just show a blank screen which confused me (on stub)
> 
> Yes, this can happen as new courthouses can be created, or all users could be removed from a courthouse.

Let me guess though, there's no design for what it should look like with no users?

File: src/app/core/components/common/filters/filters.component.html [hunk 1]
```diff
@@ -49,44 +54,59 @@ <h3 class="govuk-heading-s govuk-!-margin-bottom-0">{{ selected.displayName }}</
...
+                  class="moj-pagination__link"
+                  [class.flip]="!filter.isCollapsed"
+                  href="javascript:void(0)"
+                  (click)="filter.isCollapsed = !filter.isCollapsed"
+                  ><span class="govuk-visually-hidden"> Expand {{ filter.displayName }}</span></a
+                >
+              </div>
+              <label [for]="'search-filter-' + index">{{ filter.displayName }}</label>
+            </legend>
+            @if (filter.isCollapsed) {
```
Comment (DavidTMann) [inline src/app/core/components/common/filters/filters.component.html:72]: I think isCollapsed is named incorrectly or needs to be set appropriately as currently it shows those items when expanded not collapsed

**Review by DavidTMann — CHANGES_REQUESTED** [review]

**Review by DavidTMann — APPROVED** [review]

**Pull request merged**